doc/lightning-close.7.txt

```diff
@@ -8,36 +8,38 @@ lightning-close - Command for closing channels with direct peers
 
 SYNOPSIS
 --------
-*close* 'id' ['force'] ['timeout']
+*close* 'id' ['unilateraltimeout']
 
 DESCRIPTION
 -----------
 
 The *close* RPC command attempts to close the channel cooperatively
-with the peer.
+with the peer, or unilaterally after 'unilateraltimeout' (unless 0).
 If the given 'id' is a peer ID (66 hex digits as a string), then
 it applies to the active channel of the direct peer corresponding to
 the given peer ID.
 If the given 'id' is a channel ID (64 hex digits as a string, or
 the short channel ID 'blockheight:txindex:outindex' form), then it
 applies to that channel.
 
-The *close* command will time out and return with an error when the
-number of seconds specified in 'timeout' is reached.
-If unspecified, it times out in 30 seconds.
+The *close* command will unilaterally close the channel when the
+number of seconds specified by 'unilateraltimeout' is reached,
+unless that is 0.  The default is 2 days (172800 seconds).
```

Comment:
Suggest describing in more detail what happens if 'unilateraltimeout' is 0.

    If 'unilateraltimeout' is 0, then the *close* command will wait
    indefinitely until the peer is online and can negotiate a
    mutual close.